tests/conftest.py

```diff
@@ -156,6 +161,11 @@ def mock_cluster_resp_slaves(request, **kwargs):
     return _gen_cluster_mock_resp(r, response)
 
 
+@pytest.fixture(scope="module")
```

Comment:
Does the `master_host` value ever change from module to module? If not, scoping this to `'session'` seems more appropriate.

Reply:
It shouldn't, no - updated!